packages/ERTP/src/types.js

```diff
@@ -47,23 +47,78 @@
  * @returns {Amount}
  */
 
+/**
+ * This section blindly imitates what Endo's ses/src/error/types.js
+ * does to express type overloaded methods.
+ *
+ * @callback AmountMakeBrandValue
+ * @param {Brand} brand
+ * @param {Value} allegedValue
+ * @returns {Amount}
+ *
+ * @callback AmountMakeValueBrand
```

Comment:
You can mark this as deprecated, I think, since it's a separate function

Reply:
Adding an `@deprecated` above line 59 doesn't seem to do anything. vscode doesn't show me any strikethoughs. Adding with TODO comments

Reply:
Ah, I think you have to add it to type above the function itself. But the function is the combination of the two types and shouldn't be deprecated. I withdraw the comment that you can add `@deprecated`